lib/model.js

```diff
@@ -3028,6 +3028,9 @@ Model.aggregate = function aggregate(pipeline, callback) {
   return aggregate;
 };
 
+Model.popregate = function popugrate() {
```

Comment:
Is this here intentionally or is this a typo?

Reply:
In my spare time I have working on trying to make a version of populate that uses aggregate. That lame attempt at a name for what I've been working on should have been in another branch. My bad. I'll remove it and re-commit.